quote.py

```diff
@@ -3,48 +3,102 @@
 import random
 from halibot import HalModule
 
+sr = random.SystemRandom()
+
 class Quote(HalModule):
 
-	quotes = []
+	# Configuration defaults
+	remove_votes = 3
+	max_history = 10
```

Comment:
Agreed with @sjrct, but keep the deletion of the `quotes = []`, that shouldn't be there anymore anyway